### PR TITLE
Installs docker Version 1.13

### PR DESCRIPTION
**Pull request by nranjanShippable**
https://github.com/Shippable/admiral/issues/987

Tested with installing in AWS with docker version 1.12 and 1.13+.

### CODE REVIEW

File: admiral.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ readonly AWSCLI_VERSION=1.10.63
 readonly API_TIMEOUT=600
 readonly ADMIRAL_PORT=50003
 export LC_ALL=C
+export docker_restart=false
```
Comment (ric03uec) [inline admiral.sh]: why do we need this?

Comment (nranjanShippable) [inline admiral.sh]: When we change docker options, to add shippable docker options. We need to restart docker after that, so this acts as a flag.

**Comment by nranjanShippable** [comment]
@ric03uec Right there is a difference with the existing script that here we are not getting static docker binaries.

File: admiral.sh (hunk 1 above)
Comment (ric03uec) [inline admiral.sh]: i dont think we need to do this for local installation. once docker boots up, everything else should just work without changes. Setting docker opts and restarting is unnecessary. unless im missing something here?

**Pull request merged**